package.json

```diff
@@ -125,7 +125,7 @@
     "lint:changelog": "node ./scripts/lint-changelog.js",
     "lint:prettier": "prettier \"**/*.{md,json,yml,html,css}\" --check",
     "lint:dist": "eslint --no-eslintrc --no-ignore --env=es6,browser --parser-options=ecmaVersion:2016 \"dist/!(bin-prettier|index|third-party).js\"",
-    "lint:spellcheck": "cspell {bin,scripts,src,website}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md",
+    "lint:spellcheck": "cspell \"**/*\" \".github/**/*\"",
```

Comment:
Is `.github/**/*` still needed after you added the quotes around `**/*`?

Reply:
Yep, still not checking

```text
node -p "require('./cspell.json').words.find(x => x.toLowerCase() === 'macos')" && yarn lint:spellcheck
undefined
yarn run v1.22.4
$ cspell "**/*"
CSpell: Files checked: 1450, Issues found: 0 in 0 files
Done in 30.45s.
```